plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/style.scss

```
@@ -229,7 +229,7 @@
 			justify-content: flex-start;
 			border-width: 0;
 		}
-		.wc-block-components-radio-control__label img {
+		.wc-block-components-radio-control__label > img {
```

Let's add this comment for more clarity
```suggestion
        // Targets payment card icons used in payment options
		.wc-block-components-radio-control__label > img {
```

---

Could you commit this suggestion before merging?

---

Yes :)